eng/pipelines/client.yml

```diff
@@ -2,16 +2,16 @@
 # This file is generated automatically and any changes will be lost.
 
 pr:
-  branches:
-    include:
-    - users/mitchdenny/pr-pipelines
   paths:
     include: # Changes in these directories cause us to build everything for validation purpoes.
     - eng/
     - .azure-pipelines/
 
+trigger:
```

Comment:
Did you ever figure out whether or not master is the default trigger or if it is all?